packages/headless/src/controllers/commerce/product-view/headless-product-view.test.ts

```diff
@@ -0,0 +1,36 @@
+import {ProductRecommendation} from '../../../api/search/search/product-recommendation';
+import {buildMockCommerceState} from '../../../test/mock-commerce-state';
+import {
+  buildMockCommerceEngine,
+  MockedCommerceEngine,
+} from '../../../test/mock-engine-v2';
+import {buildProductView, ProductView} from './headless-product-view';
+
+
+describe('Productview', () => {
```

Comment:
Nit: would camel-casing be more consistent?
```suggestion
describe('ProductView', () => {
```